inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -0,0 +1,46 @@
+"""Class for the coils creation."""
+
+class StellaratorCoils:
```

Comment:
If this is supposed to be a "scenario", I am not 100% sure that StelleratorCoils is a good name for the class. This is not important for now, but I think it would be beneficial to have a name that invokes the idea of a process, i.e of something that is going to be simulated.

Reply:
Ok, I will take note of this for now. I will try to come up with a better name for the class.